### ISSUE TITLE
regression fails with java.nio.file.FileSystemException: /home/jenkins/root/workspace/gh_centos7-regression/builder210.int.aws.gluster.org:/d/backends/patchy0: Operation not permitted

### ISSUE THREAD

**Issue by itisravi** [gluster/project-infrastructure#93]
Latest run:https://build.gluster.org/job/gh_centos7-regression/70/console
I guess @deepshikhaaa already knows about this. I am creating an issue so that we can track it.

**Comment by deepshikhaaa** [comment]
It should have been fixed now.

**Comment by itisravi** [comment]
This is still happening: https://build.gluster.org/job/gh_centos7-regression/136/console

**Comment by deepshikhaaa** [comment]
1) There is some change in a particular test which is not cleaning up the patchy/pycache directories/files. They are created when we run regression test suite. It should be deleted as a part of the cleanup after every test is run. We couldn't identify which patch has introduced this. Because of these leftovers(last build leftovers), the Jenkins is not able to clean the workspace directory before any build starts. So if someone can identify the culprit patch?

2) To fix this up, we did add workspace cleanup steps when the build finishes. It should have fixed the issue but the cleanup script(https://github.com/gluster/glusterfs-patch-acceptance-tests/blob/master/cleanup.sh) was aborting the script at a step which `pkills` the processes when running the regression. And so the new bug. Another hack was to add the `rm -rf $WORKSPACE` explicitly in the end of build script but with this the artifacts/logs tar was also getting deleted with deleting the workspace directory.

Now, I have changed the ownership of everything/leftovers in the $WORKSPACE. it would allow the jenkins user to wipe out before the build starts. Lets see if it fixes the issue.
